cmd/vic-machine/delete/delete.go

```diff
@@ -142,15 +148,15 @@ func (d *Uninstall) Run(clic *cli.Context) (err error) {
 	if err != nil {
 		log.Error("Failed to get Virtual Container Host configuration")
 		log.Error(err)
-		return err
+		return errors.New("Delete failed")
 	}
 
 	// compare vch version and vic-machine version
 	installerBuild := version.GetBuild()
 	if vchConfig.Version == nil || !installerBuild.Equal(vchConfig.Version) {
 		if !d.Data.Force {
 			log.Errorf("VCH version %q is different than installer version %s. Specify --force to force delete", vchConfig.Version.ShortVersion(), installerBuild.ShortVersion())
-			return errors.New("delete failed")
+			return errors.New("Delete failed")
```

Comment:
I suspect it should be lowercase "delete" since you use lowercased error in other places.